spyder_unittesting/widgets/unittestinggui.py

```diff
@@ -166,9 +150,7 @@ def __init__(self, parent):
                            self.start_button, self.stop_button, browse_button,
                            self.collapse_button, self.expand_button):
                 widget.setDisabled(True)
-            text = _(
-                '<b>Please install the <a href="%s">line_profiler module</a></b>'
-                ) % WEBSITE_URL
+            text = _('<b>Please install the unittesting module</b>')
```

Comment:
It should not be unittesting here but pytest I think. Again, this was not finished...